### ISSUE TITLE
icetray::I3Module printf->log_xxx (Trac #1657)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1649]
put this message to logging stream, how it should be:
log_info or log_notice

http://code.icecube.wisc.edu/projects/icecube/browser/IceTray/projects/icetray/trunk/private/icetray/I3Module.cxx#L82

Migrated from https://code.icecube.wisc.edu/ticket/1657
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T20:45:45", 
    "description": "put this message to logging stream, how it should be:\nlog_info or log_notice\n\nhttp://code.icecube.wisc.edu/projects/icecube/browser/IceTray/projects/icetray/trunk/private/icetray/I3Module.cxx#L82", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547239545569546", 
    "component": "other", 
    "summary": "icetray::I3Module printf->log_xxx", 
    "priority": "normal", 
    "keywords": "icetray logging", 
    "time": "2016-04-26T12:29:41", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160426T14:57:08: 
* olivas changed status from "new" to "closed"
* olivas commented:

> In changeset 2524:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2524"
> replace printf with log_info.  closes #1649.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:45:45: nwhitehorn commented:

> In changeset 168726:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168726"
> replace printf with log_info.  closes #1649.
> ```
